components/proxy/README.md

```diff
@@ -1,3 +1,3 @@
 # *Quality-time* proxy
 
-Wrap the Caddy image in a *Quality-time* image so the Caddy version number can be changed when needed.
\ No newline at end of file
+See the [software documentation](https://quality-time.readthedocs.io/en/latest/software.html#proxy).
```

Comment:
On the other hand, I think this README, as well as the one for the renderer might aswell be deleted... The linked information is only a single line.

Reply:
Yeah, but it is a deep link to the documentation about the component, in this case the proxy, so that if you're browsing the source code it's easy to open the accompanying docs. Admittedly, not as easy as having the docs right there...